recipe/patches/0003-set-cmake-paths.patch

```diff
@@ -1,31 +0,0 @@
-From 40978d5b00896f0941802f565887823f27000a34 Mon Sep 17 00:00:00 2001
-From: Isuru Fernando <isuruf@gmail.com>
-Date: Wed, 14 Oct 2020 20:06:05 -0500
-Subject: [PATCH 3/3] set cmake paths
```

Comment:
This patch doesn't apply anymore after https://github.com/llvm/llvm-project/commit/23c665371a98c7a16765d12bf2db19b0626b7181, and it looks to me like it's not needed anymore either.